.circleci/config.yml

```
@@ -58,3 +58,15 @@ workflows:
             tags:
               only: /^v.*/
 
+      - architect/integration-test:
+          context: architect
+          name: admission-integration-test
+          setup-script: "integration/test/organization/setup.sh"
```

```suggestion
          install-app-platform: true
```

If you use the latest architect-orb (0.17.0) then you don't need the setup script.

We need docs for that. I'll work on them this week.